src/v0/destinations/facebook_pixel/networkHandler.js

```diff
@@ -117,7 +117,7 @@ const destResponseHandler = (destinationResponse) => {
 
 const networkHandler = function () {
   // The order of execution also happens in this way
-  this.prepareProxyRequest = prepareProxyRequest;
+  this.prepareProxy = prepareProxyRequest;
```

Comment:
Don't we need to update the name from where this networkHandler is getting called?

Reply:
No, it is not needed